StateWrappers.as

```diff
@@ -1,4 +1,25 @@
-#if MP4
+#if TMNEXT
+
+namespace VehicleState
+{
+	shared enum FallingState {
+		FallingAir = 0,
+		FallingWater = 2,
+		RestingGround = 4,
+		RestingWater = 6
+	}
```

Comment:
These values are kind of arbitrary - do you know what 1, 3, 5 means?

Reply:
I didn't find any case where this was set to an odd value, so I'm assuming it's only ever even. Not sure why

Reply:
Generally, I'm not comfortable merging something w/o really knowing what it does. These values are not very complete and I can't find any enum in the game's exposed enums that would match these values either.

However, if this is something people really need in the API, we can think about just putting it in with some disclaimers in the comments that it might break.

Reply:
Understood. For each wheel, this is the final value before the next wheel starts, so really I just want them for the sake of completeness. My theory is that this is used to calculate some things in `CSmScriptPlayer` like `WheelsContactCount`